### PR TITLE
Adds jani HUDs from Baystation!

### PR DESCRIPTION
**Pull request by Henri215**
## What Does This PR Do

From https://github.com/Baystation12/Baystation12/pull/22788.
Adds the janitor HUD (base, sunglasses and night vision versions), used to clearly see filth tiles.

Added to cyborgs, implant version, research design, janitor ERT.

## Why It's Good For The Game

It is another thing to make janitors feel badass and that helps you in locating where the filth tiles are.
## Images of changes

https://github.com/ParadiseSS13/Paradise/assets/77684085/bab6d83b-f398-4bb9-b854-a0c52b24c127

## Testing

- Used the hud as a cyborg
- Used all 3 different glasses to test them
- Verified vendor and protolathe
## Changelog
:cl:
add: Added the janitor HUD, available at protolathes
/:cl:

### CODE REVIEW

File: code/modules/response_team/ert_outfits.dm [hunk 1]
```diff
@@ -499,15 +499,15 @@
...
+		/obj/item/push_broom = 1,
 		/obj/item/storage/box/lights/mixed = 1,
 		/obj/item/melee/flyswatter = 1)
 
 /datum/outfit/job/centcom/response_team/janitorial/amber
 	name = "RT Janitor (Amber)"
 	suit = /obj/item/clothing/suit/armor/vest/ert/janitor
 	head = /obj/item/clothing/head/helmet/ert/janitor
-	glasses = /obj/item/clothing/glasses/sunglasses
+	glasses = /obj/item/clothing/glasses/hud/janitor/sunglasses
```
Comment (Contrabang) [inline code/modules/response_team/ert_outfits.dm:510]: Could you please give all these guys sec-hud implants?

File: code/game/data_huds.dm [hunk 2]
```diff
@@ -487,6 +490,14 @@
...
 		holder.icon_state = ""
 
+/*~~~~~~~~~~~~~~
+	JANI HUD
+~~~~~~~~~~~~~~~*/
+/obj/effect/decal/cleanable/proc/jani_hud_set_sign()
+	var/image/holder = hud_list[JANI_HUD]
+	holder.icon_state = "caution"
+	holder.plane = ABOVE_LIGHTING_PLANE
+
```
Comment (ppi13) [inline code/game/data_huds.dm:500]: Can you center the icon?

Comment (Henri215) [reply code/game/data_huds.dm:500]: Are you sure about that? It would block vision of what the cleanable is in some cases.

Comment (ppi13) [reply code/game/data_huds.dm:500]: You can add `mouse_opacity = MOUSE_OPACITY_TRANSPARENT` and icon size, shape and location is irrelevant.

**Comment by McRamon** [comment]
hm i dont like how this looks
ikk think about improving it

File: code/modules/response_team/ert_outfits.dm (hunk 1 above)
Comment (Henri215) [reply code/modules/response_team/ert_outfits.dm:510]: Amber, red and gamma janitors? We want all ERT loadouts to have access to sec hud?

Comment (Contrabang) [reply code/modules/response_team/ert_outfits.dm:510]: Yeah, I'd say all of them just to not make this an ERT balance change

Comment (Henri215) [reply code/modules/response_team/ert_outfits.dm:510]: Amber = sunglasses
Red = sec sunglasses
Gamma = nightvision goggles
Thats what they had, giving them all sec-hud implants would be an ERT balance change.

Comment (Contrabang) [reply code/modules/response_team/ert_outfits.dm:510]: Eh you're right, just give it to what ones had it before then I guess

File: code/game/data_huds.dm (hunk 2 above)
Comment (Henri215) [reply code/game/data_huds.dm:500]: image
Something like this?

File: code/modules/response_team/ert_outfits.dm (hunk 1 above)
Comment (Henri215) [reply code/modules/response_team/ert_outfits.dm:510]: Done in next commit (why is gamma engineer the only non sec themed ERT that gets a sec-hud implant????)

File: code/game/data_huds.dm (hunk 2 above)
Comment (ppi13) [reply code/game/data_huds.dm:500]: Yeah, this looks far better! Looking forward to the updated Icon!

**Comment by Henri215** [comment]
image
image

Updated glasses sprites

**Comment by Henri215** [comment]
image

I 200% didnt touch these, no idea why it appears as modified

**Comment by Henri215** [comment]
Draft until sprites are final

File: code/modules/research/designs/janitorial_designs.dm [hunk 3]
```diff
@@ -60,3 +60,13 @@
...
+
+/datum/design/light_replacer_bluespace
+	name = "Filth Scanner"
+	desc = "A heads-up display that scans for messes and alerts the user. Good for finding puddles hiding under catwalks."
+	id = "janitor_hud"
+	req_tech = list("biotech" = 1, "magnets" = 3)
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 500, MAT_GLASS = 500)
+	build_path = /obj/item/clothing/glasses/hud/janitor
+	category = list("Janitorial")
```
Comment (DGamerL) [inline code/modules/research/designs/janitorial_designs.dm:72]: Copypaste oops?

**Review by DGamerL — COMMENTED** [review]
1. Not sure on the name, especially because you call the eye-implant a Janitor HUD. Please choose either one for consistency.
2. See my discord messages

File: code/modules/research/designs/janitorial_designs.dm (hunk 3 above)
Comment (Henri215) [reply code/modules/research/designs/janitorial_designs.dm:72]: Absolutely not :D, next commit.

File: code/modules/research/designs/janitorial_designs.dm [hunk 4]
```diff
@@ -60,3 +60,13 @@
...
+
+/datum/design/janitor_hud
+	name = "Janitor HUD"
+	desc = "A heads-up display that scans for messes and alerts the user. Good for finding puddles hiding under catwalks."
+	id = "jani_hud"
+	req_tech = list("biotech" = 1, "magnets" = 3)
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 500, MAT_GLASS = 500)
+	build_path = /obj/item/clothing/glasses/hud/janitor
+	category = list("Janitorial")
```
Comment (DGamerL) [inline code/modules/research/designs/janitorial_designs.dm:72]: As spoken about in the discord, please fix this

File: code/modules/research/designs/medical_designs.dm [hunk 5]
```diff
@@ -386,6 +386,17 @@
 	build_path = /obj/item/organ/internal/cyberimp/arm/toolset_abductor
 	category = list("Medical")
 
+/datum/design/cyberimp_jani_hud
+	name = "Janitor HUD Implant"
+	desc = "These cybernetic eye implants will display a filth HUD over everything you see. Wiggle eyes to control."
+	id = "ci-janihud"
+	req_tech = list("materials" = 5, "engineering" = 4, "programming" = 4, "biotech" = 4)
+	build_type = PROTOLATHE | MECHFAB
+	construction_time = 50
```
Comment (DGamerL) [inline code/modules/research/designs/medical_designs.dm:395]: Is this in deciseconds? Could you use the define if so?

**Review by DGamerL — CHANGES_REQUESTED** [review]

**Comment by Henri215** [comment]
Basic Jani HUD removed from the vendor, considering its a upgrade it would feel more special by having to acquire it instead of being handed for "free" roundstart.

**Pull request merged**